client/src/main/java/org/apache/uniffle/client/impl/ShuffleReadClientImpl.java

```diff
@@ -101,17 +110,32 @@ public ShuffleReadClientImpl(ShuffleClientFactory.ReadClientBuilder builder) {
       builder.clientType(builder.getRssConf().get(RssClientConf.RSS_CLIENT_TYPE));
     } else {
       // most for test
-      RssConf rssConf = new RssConf();
+      RssConf rssConf = (builder.getRssConf() == null) ? new RssConf() : builder.getRssConf();
       rssConf.set(RssClientConf.RSS_STORAGE_TYPE, builder.getStorageType());
       rssConf.set(RssClientConf.RSS_INDEX_READ_LIMIT, builder.getIndexReadLimit());
       rssConf.set(
           RssClientConf.RSS_CLIENT_READ_BUFFER_SIZE, String.valueOf(builder.getReadBufferSize()));
+      if (!rssConf.contains(RssClientConf.BLOCKID_SEQUENCE_NO_BITS)) {
```

Comment:
Maybe we should add more constraints about the the bits. Such as the total bits can't exceed 64bits.